extensions/amp-bind/amp-bind.md

```diff
@@ -129,6 +131,37 @@ For performance and to avoid the risk of unexpected content jumping, `amp-bind`
 <button on="tap:AMP.setState({currentAnimal: 'cat'})">Set to Cat</button>
 ```
 
+[/filter]
+
+[filter formats="email"]
+
+```html
+<!-- Store complex nested JSON data in <amp-state> elements. -->
+<amp-state id="myAnimals">
+  <script type="application/json">
+    {
+      "dog": {
+        "imageUrl": "/img/dog.jpg",
+        "style": "greenBackground"
+      },
+      "cat": {
+        "imageUrl": "/img/cat.jpg",
+        "style": "redBackground"
+      }
+    }
+  </script>
+</amp-state>
+
+<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+
+<!-- CSS classes can also be added or removed with [class]. -->
+<p class="greenBackground" [class]="myAnimals[currentAnimal].style">
+  Each animal has a different background color.
+</p>
+```
+
+[/filter]
```

Comment:
Can we just put a filter around the `[src]` binding usage on lines 122 to 129 instead?

Reply:
I'm unsure if the filtering system will work inside of a code sample. Ideally we're moving towards samples being inline executable as well... @sebastianbenz for any input?